docs/CONTRIBUTING.rst

```diff
@@ -2,111 +2,75 @@
-* `Suggest new data and features <https://github.com/catalyst-cooperative/pudl/issues/new?template=feature_request.md>`__ that would be useful.
+  discussion board
+* Tell us about what data you're looking for by opening an `issue
+  <https://github.com/catalyst-cooperative/pudl/issues/new?assignees=&labels=new-data&projects=&template=new_dataset.md&title=>`__
+* Tell us what you're trying to do with PUDL data in `this thread
+  <https://github.com/orgs/catalyst-cooperative/discussions/3105>`__
 * `File bug reports <https://github.com/catalyst-cooperative/pudl/issues/new?template=bug_report.md>`__ on Github.
-* Help expand and improve the documentation, or create new
-  `example notebooks <https://github.com/catalyst-cooperative/pudl-examples/>`__
-* Help us create more and better software :doc:`test cases <dev/testing>`.
-* Give us feedback on overall usability using `GitHub Discussions
+* Tell us what you'd like to see in PUDL in the `Ideas
   <https://github.com/orgs/catalyst-cooperative/discussions/categories/ideas>`__
-  -- what's confusing?
-* Tell us a story about how you're using of the data.
-* Point us at interesting publications related to open energy data, open source energy
-  system modeling, how energy policy can be affected by better data, or open source
-  tools we should check out.
-* Cite PUDL using
-  `DOIs from Zenodo <https://zenodo.org/communities/catalyst-cooperative/>`__
-  if you use the software or data in your own published work.
+  discussion board
+
+.. _code-contribs:
+
+--------------------
+Code contributions
+--------------------
+
+.. include:: ../CONTRIBUTING.rst
+   :start-after: after-intro:
+
+.. _connect-orgs:
+
+-----------------------------------
+Connect us with other organizations
+-----------------------------------
+
+For PUDL to make a bigger impact, we need to find more people who need the data.
+Here's how you can help:
+
+* Cite PUDL using `DOIs from Zenodo
+  <https://zenodo.org/communities/catalyst-cooperative/>`__ if you use the
+  software or data in your own published work.
 * Point us toward appropriate grant funding opportunities and meetings where
   we might present our work.
+* Point us at interesting publications related to open energy data, open source
+  energy system modeling, how energy policy can be affected by better data, or
+  open source tools we should check out.
 * Share your Jupyter notebooks and other analyses that use PUDL.
 * `Hire Catalyst <https://catalyst.coop/hire-catalyst/>`__ to do analysis for
   your organization using the PUDL data -- contract work helps us self-fund
   ongoing open source development.
-* Contribute code via
-  `pull requests <https://help.github.com/en/articles/about-pull-requests>`__.
-  See the :doc:`developer setup <dev/dev_setup>` for more details.
-* And of course... we also appreciate
-  `financial contributions <https://www.paypal.com/cgi-bin/webscr?cmd=_s-xclick&hosted_button_id=PZBZDFNKBJW5E&source=url>`__.
```

Comment:
Does this get mentioned elsewhere? I don't want this to get lost.

Reply:
I'll add it back in. I think the donations are never gonna be a huge source but no harm in calling it out :)